aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/AWSCognitoAuthPlugin.java

```diff
@@ -117,6 +117,7 @@
     private static final long SECONDS_BEFORE_TIMEOUT = 10;
     private static final String COGNITO_USER_ID_ATTRIBUTE = "sub";
     private static final String MOBILE_CLIENT_TOKEN_KEY = "token";
+    private static final String EMPTY_STRING = "";
```

Comment:
Per https://github.com/aws-amplify/amplify-android/pull/763#issuecomment-684967095, the current way to specify no password is to pass null, instead of empty string.    Is that incorrect?

Reply:
Saijad noticed that the lambda that is generated by CLI to handle custom auth will only work with empty string (and break with `null`)